client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/types.ts

```diff
@@ -12,7 +15,8 @@ export interface NavigatorLocation {
 }
 
 export type Category = {
-	name: string;
-	label: string;
-	description: string;
+	name?: string;
+	slug?: string;
+	label?: string;
+	description?: string;
 };
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)
Each property is optional here, but in what case is the data can be null? 👀

Reply:
I did that because in the static file, the category is hardcoded, but only the slug is required.